plugins/modules/consul_policy.py

```diff
@@ -135,22 +103,11 @@
 """
 
 from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.general.plugins.module_utils.consul import (
+    _ConsulModule, auth_argument_spec)
```

Comment:
Here you could also do
```suggestion
    _ConsulModule as ConsulModule, auth_argument_spec)
```
to avoid propagating the rename to the modules, but let's keep it at is now :)